File: JavaScript/ordenamientoSeleccion.js
function seleccion() {
    let arreglo = new Array(10);
    let minimo, aux, posmin;

    // Llenado con números aleatorios entre 0 y 100
    for (let i = 0; i < arreglo.length; i++) {
        arreglo[i] = Math.floor(Math.random() * 101);
    }

    console.log("Arreglo antes de ordenar:");
    console.log(arreglo.join(" "));

    // Algoritmo de selección
    for (let i = 0; i < arreglo.length; i++) {
        minimo = arreglo[i];
        posmin = i;

        for (let j = i + 1; j < arreglo.length; j++) {
            if (arreglo[j] < minimo) {
                minimo = arreglo[j];
                posmin = j;
            }
        }

        aux = arreglo[i];
        arreglo[i] = arreglo[posmin];
        arreglo[posmin] = aux;
    }

    console.log("Arreglo ordenado:");
    console.log(arreglo.join(" "));
}

seleccion();

File: Python/ordenamientoSeleccion.py
import random

def seleccion():
    arreglo = [random.randint(0, 100) for _ in range(10)]
    
    print("Arreglo antes de ordenar:")
    print(" ".join(map(str, arreglo)))

    # Algoritmo de selección
    for i in range(len(arreglo)):
        minimo = arreglo[i]
        posmin = i

        for j in range(i + 1, len(arreglo)):
            if arreglo[j] < minimo:
                minimo = arreglo[j]
                posmin = j

        aux = arreglo[i]
        arreglo[i] = arreglo[posmin]
        arreglo[posmin] = aux

    print("Arreglo ordenado:")
    print(" ".join(map(str, arreglo)))


seleccion()
